### PR TITLE
chore(demo): site fixes based on UX feedback

### LINKED ISSUE #269: chore(demo): site fixes based on UX feedback
- [x] blue is too blue
- [x] different jumbotrons for landing vs. docs pages (smaller for docs)
- [x] less text in docs jumbotron
- [x] size of "Code on GH" button is too big
- [x] add navigation to top
- [x] unify all the blues

### PR DESCRIPTION
**Pull request by icfantv**
* update demo site to address UX issues raised
* update docs only as necessary to address above

fixes #269

### CODE REVIEW

File: src/dropdown/dropdown.ts [hunk 1]
```diff
@@ -1,11 +1,24 @@
-import {Directive, Input, HostListener} from '@angular/core';
+import {Directive, ElementRef, Input, HostListener} from '@angular/core';
```
Comment (icfantv) [inline src/dropdown/dropdown.ts]: Just a note that I changed this file so I could do some testing as the dropdown does not close when you click on a menu item nor does it close when you click outside the menu on the rest of the document.  Feel free to reject the PR if this is totally unwanted in the library and I will remove.  Just know the menu UX is very poor in the interim.

**Comment by icfantv** [comment]
@Foxandxss is fixing the build - typings didn't get installed.

**Comment by icfantv** [comment]
Reverted change to dropdown....

File: demo/src/app/home/home.component.html [hunk 2]
```diff
@@ -1,4 +1,25 @@
...
+<div class="jumbotron">
+  <div class="container">
+    <!--<p class="github-buttons pull-lg-right">-->
+      <!--<a class="github-button" href="https://github.com/ng-bootstrap/core" data-icon="octicon-star" data-style="mega" data-count-href="/ng-bootstrap/core/stargazers" data-count-api="/repos/ng-bootstrap/core#stargazers_count"-->
+         <!--data-count-aria-label="# stargazers on GitHub" aria-label="Star ng-bootstrap/core on GitHub">Star</a>-->
+      <!--<a class="github-button" href="https://github.com/ng-bootstrap/core/fork" data-icon="octicon-repo-forked" data-style="mega" data-count-href="/ng-bootstrap/core/network" data-count-api="/repos/ng-bootstrap/core#forks_count"-->
+         <!--data-count-aria-label="# forks on GitHub" aria-label="Fork ng-bootstrap/core on GitHub">Fork</a>-->
+      <!--<a class="github-button" href="https://github.com/ng-bootstrap/core" data-icon="octicon-eye" data-style="mega" data-count-href="/ng-bootstrap/core/watchers" data-count-api="/repos/ng-bootstrap/core#subscribers_count"-->
+         <!--data-count-aria-label="# watchers on GitHub" aria-label="Watch ng-bootstrap/core on GitHub">Watch</a>-->
+    <!--</p>-->
```
Comment (pkozlowski-opensource) [inline demo/src/app/home/home.component.html]: Why is this part commented? Is it temporary? If not let's remove this code.

File: demo/src/public/index.html [hunk 3]
```diff
@@ -4,7 +4,7 @@
     <meta charset="utf-8">
     <title>ng-bootstrap Demo</title>
     <link rel="shortcut icon" type="image/x-icon" href="/img/favicon.ico">
-    <script async defer id="github-bjs" src="https://buttons.github.io/buttons.js"></script>
+    <!--<script async defer id="github-bjs" src="https://buttons.github.io/buttons.js"></script>-->
     <base href="/">
```
Comment (pkozlowski-opensource) [inline demo/src/public/index.html]: Why this part is commented out?

**Comment by pkozlowski-opensource** [comment]
@icfantv I don't think we are in a better shape with the proposed changes, sorry :-/

There are number of issues with the new proposal:
* we shouldn't remove side-nav (it is fine to keep it _in addition_ to top-nav)
* we shouldn't have drop-down with Directives - just "Directives" link
* not sure what the "Previous docs" link is supposed to do - is it for the future?
* I think that we have misunderstood each other regarding GitHub buttons. What I was getting at is that the "Code on GitHub" button is too big (maybe we shouldn't have it at all?)

All in all I don't think we want to land it in the current form. As an alternative we can do 2 things:
* you can send another proposal
* we can have a hangout together and work interactively on this
* I can send a proposal with changes and you can take it from there.

Let me know what works best for you.

**Comment by icfantv** [comment]
@pkozlowski-opensource, @Foxandxss updated per meeting and comments.  using the multi-slot routing functionality was broached by Jesús but after seeing the URL syntax, I'm not so sure...

It would, however, buy us some DRY across files...

**Comment by pkozlowski-opensource** [comment]
@icfantv unfortunately it doesn't merge cleanly after I've landed #259. Could you please rebsae?

**Comment by Foxandxss** [comment]
Rebase it and I want to review it before we merge it.

**Comment by icfantv** [comment]
@pkozlowski-opensource i'm on it.  @Foxandxss, will ping when updated.

**Comment by icfantv** [comment]
@pkozlowski-opensource, @Foxandxss, ok, this is rebased w/ master and alert is merged.

**Pull request closed without merging**